pkgs/development/python-modules/diff_cover/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, buildPythonPackage, fetchFromGitHub, jinja2, jinja2_pluralize, pygments,
+  six, inflect, mock, nose, coverage, pycodestyle, flake8, pyflakes, git,
+  pylint, pydocstyle }:
+
+buildPythonPackage rec {
```

Comment:
Does this tool need to access Python modules of the environment it is used in, or is it just an external tool?

Reply:
( Maybe I didn't understand what you ask. I'm learning english ... :) )
This tool is not supposed to provide python modules for users.
It provide two execution files: `diff-cover`, `diff-quality`

Reply:
This tool doesn't need access to environment where it is used in.